dbt_subprojects/tokens/models/tokens/sonic/tokens_sonic_erc20.sql

```diff
@@ -50,12 +50,19 @@ FROM (VALUES
     , (0xD31E89Ffb929b38bA60D1c7dBeB68c7712EAAb0a, 'bSolvBTC-13', 18)
     , (0x7870ddFd5ACA4E977B2287e9A212bcbe8FC4135a, 'BeefySiloV2USDC.e', 6)
     , (0xe51ee9868c1f0d6cd968a8b8c8376dc2991bfe44, 'BRUSH', 18)
+    , (0x9fb76f7ce5FCeAA2C42887ff441D46095E494206, 'wstkscUSD', 6)
+    , (0x0C4E186Eae8aCAA7F7de1315D5AD174BE39Ec987, 'anS', 18)
```

Comment:
Seems this line is duped:
https://github.com/duneanalytics/spellbook/actions/runs/13372061386/job/37342670207?pr=7541#step:13:47

Reply:
interesting that it didn't fail prod 🤔 
i wonder if the token is in automated source, so it's ignored in manual here. let me dig in a bit and see what i can do.